killbill-integration-tests/core/test_entitlement_add_on.rb

```diff
@@ -170,22 +170,22 @@ def test_cancel_bp_with_cancel_date_and_uncancel
       assert_equal('START_ENTITLEMENT', bp.events[0].event_type)
       assert_equal('sports-monthly', bp.events[0].plan)
       assert_equal('sports-monthly-trial', bp.events[0].phase)
-      assert_equal('2013-08-01', bp.events[0].effective_date)
+      assert_equal('2013-08-01', bp.events[0].effective_date.partition('T')[0]) if bp.events[0].effective_date
```

Comment:
Maybe we should introduce a method (something like `assert_equal_dates`) to abstract that logic?

```
assert_equal_dates('2013-08-01', bp.events[0].effective_date)

def assert_equal_dates(expected, actual)
  if actual.nil?
    assert_nil(expected)
  else
    assert_equal(expected, actual.partition('T')[0])
  end
end
```

This could go with the other entitlement checkers?

Reply:
Fixed in https://github.com/killbill/killbill-integration-tests/pull/51.